design-proposals/klip-42-schema-migrations-tool.md

```diff
@@ -239,11 +244,9 @@ The below schema is designed so we support undo operations in the future. When a
 
 This is the `CREATE` statement for the `SCHEMA_VERSION` table:
 ```sql
-CREATE TABLE schema_version
+CREATE TABLE schema_version /* name will be configurable */
   WITH (
-    KAFKA_TOPIC='default_ksql_schema_version',
-    VALUE_FORMAT='JSON',
```

Comment:
These extra configs are not necessary since they're inherited from the source by default. Also, replicas should be specified / inherited too.